package.json

```diff
@@ -22,8 +22,8 @@
   },
   "homepage": "https://github.com/zeroseven/react-screen-orientation#readme",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-rc.1",
-    "webpack": "^1.12.2"
+    "react": "^16.0.0",
+    "webpack": "^4.1.1"
```

Comment:
Not sure about jumping to webpack@4, is a big jump from 1. Maybe a progressive upgrade, using webpack@2, or no webpack upgrade at all and leaving it for another PR.

Reply:
Should webpack even be a peer dependency?